core/controllers/dashboard_test.py

```diff
@@ -389,30 +416,204 @@ def test_stats_for_multiple_explorations_with_multiple_owners(self):
         self._record_start(exp_id_1, exp_version, state_1)
         self._record_start(exp_id_2, exp_version, state_2)
         self._record_start(exp_id_2, exp_version, state_2)
+        self._record_start(exp_id_2, exp_version, state_2)
         self._run_stats_aggregator_jobs()
 
-        self._rate_exploration(exp_id_1, [3])
-        self._rate_exploration(exp_id_2, [2])
+        self._rate_exploration(exp_id_1, [5, 3])
 
         self._run_user_stats_aggregator_job()
         user_model_2 = user_models.UserStatsModel.get(self.owner_id_2)
-        self.assertEquals(user_model_2.total_plays, 4)
+        self.assertEquals(user_model_2.total_plays, 5)
         self.assertEquals(
             user_model_2.impact_score, self.USER_IMPACT_SCORE_DEFAULT)
-        self.assertEquals(user_model_2.average_ratings, 2.5)
+        self.assertEquals(user_model_2.num_ratings, 2)
+        self.assertEquals(user_model_2.average_ratings, 4)
         self.logout()
 
         self.login(self.OWNER_EMAIL_1)
         response = self.get_json(feconf.DASHBOARD_DATA_URL)
         self.assertEqual(len(response['explorations_list']), 2)
 
         user_model_1 = user_models.UserStatsModel.get(self.owner_id_1)
-        self.assertEquals(user_model_1.total_plays, 4)
+        self.assertEquals(user_model_1.total_plays, 5)
         self.assertEquals(
             user_model_1.impact_score, self.USER_IMPACT_SCORE_DEFAULT)
-        self.assertEquals(user_model_1.average_ratings, 2.5)
+        self.assertEquals(user_model_1.num_ratings, 2)
+        self.assertEquals(user_model_1.average_ratings, 4)
+        self.logout()
+
+    def test_realtime_layer_batch_job_no_ratings_plays(self):
```

Comment:
Do you need to do the DASHBOARD_DATA_URL explorations_list check? If not, maybe you could get rid of it and move these realtime tests to user_jobs_continuous_test, since that seems to be the main thing that's being tested here.